deployment/sriov-network-operator/README.md

```diff
@@ -69,6 +69,12 @@ We have introduced the following Chart parameters.
 | `operator.cniBinPath` | string | `/opt/cni/bin` | Path for CNI binary |
 | `operator.clusterType` | string | `kubernetes` | Cluster environment type |
 
+### SR-IOV Config Daemon parameters
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `sriovConfigDaemon.disablePlugins` | []string | `nil` | List of plugins to disable, currently only `mellanox` plugin is allowed |
```

Comment:
shouldn't we add "use-systemd" now that you're adding this?

Reply:
im not sure, that one can be configured during runtime.
this one is during deployment time.

Reply:
Right, didn't notice that this readme is for the helm chart